src/main/java/uk/ac/ebi/eva/pipeline/configuration/readers/VcfReaderConfiguration.java

```diff
@@ -54,11 +55,13 @@
     public VcfReader vcfReader(InputParameters parameters) throws IOException {
         String fileId = parameters.getVcfId();
         String studyId = parameters.getStudyId();
+        File vcfFile = new File(parameters.getVcf());
 
         if (VariantSource.Aggregation.NONE.equals(parameters.getVcfAggregation())) {
-            return new VcfReader(fileId, studyId, parameters.getVcf());
+            return new VcfReader(fileId, studyId, vcfFile);
         } else {
-            return new AggregatedVcfReader(fileId, studyId, parameters.getVcfAggregation(), parameters.getVcf());
+            return new AggregatedVcfReader(fileId, studyId, parameters.getVcfAggregation(),
+                    parameters.getAggregatedMappingFile(), vcfFile);
```

Comment:
This should read from default mappings in those cases where the mapping file is not specified. See the [`src/main/resources` folder from Biodata](https://github.com/EBIvariation/biodata/tree/hotfix/0.4/biodata-models/src/main/resources) for the 2 necessary mapping files, one for EXAC and one for EVS mode.